packages/react-instantsearch/src/core/createInstantSearch.js

```diff
@@ -14,15 +14,34 @@ export default function createInstantSearch(defaultAlgoliaClient, root) {
       algoliaClient: PropTypes.object,
       appId: PropTypes.string,
       apiKey: PropTypes.string,
+      children: React.PropTypes.oneOfType([
```

Comment:
because {...props}. We used to forward any props therefore the linter couldn't know about that.